lib/runner/index.js

```diff
@@ -75,17 +74,13 @@ _.assign(Runner.prototype, {
      */
     run: function (collection, options, callback) {
         var self = this,
-            runOptions = _.merge({timeout: {
-                global: DEFAULT_GLOBAL_TIMEOUT,
-                script: 0,
-                request: 0
-            }}, _.omit(options, ['environment', 'globals', 'data']), this.options.run);
+            runOptions = _.merge(_.omit(options, ['environment', 'globals', 'data']), this.options.run) || {};
```

Comment:
merge this code and the `sanitizeRunOptions` to one `this.config(options)`. 
Move `sanitizeRunOptions` as `run.prototype.config` or some other suitable name.

Parsing self's options wrt to its own options seems to me as a task for the prototype.